src/Integration/MefServices/MefSonarQubeService.cs

```diff
@@ -36,6 +40,16 @@ public class MefSonarQubeService : SonarQubeService
         public MefSonarQubeService()
             : base(new SonarQubeClient(new HttpClientHandler(), TimeSpan.FromSeconds(100)))
         {
+            // TODO: Remove the following code which only purpose is to enable a quick testing while deciding of the direction
+            // to take regarding ensuring the connection is restored on load.
+            var sqConnection = Environment.GetEnvironmentVariable("SQ_CONNECTION");
+            Debug.Assert(sqConnection != null,  "Please set up the 'SQ_CONNECTION' environment variable.");
+
+            var split = sqConnection.Split(';');
+            Debug.Assert(split.Length == 2 || split.Length == 3, $"Invalid connection information: {sqConnection}.");
+
+            var connection = new ConnectionInformation(new Uri(split[0]), split[1], split.Length == 3 ? split[2].ToSecureString() : null);
```

Comment:
I know this is just a hack and will be removed, but it would be nice if you specify the parameter names here - it is difficult to understand what is what.